code/controllers/subsystem/job.dm

```diff
@@ -605,221 +613,6 @@ SUBSYSTEM_DEF(job)
 		else //We ran out of spare locker spawns!
 			break
 
-#define TOTAL_POSITIONS "Total Positions"
-#define SPAWN_POSITIONS "Spawn Positions"
-#define PLAYTIME_REQUIREMENTS "Playtime Requirements"
-#define REQUIRED_ACCOUNT_AGE "Required Account Age"
-#define REQUIRED_CHARACTER_AGE "Required Character Age"
-
-/// Called in jobs subsystem initialize if LOAD_JOBS_FROM_TXT config flag is set: reads jobconfig.toml (or if in legacy mode, jobs.txt) to set all of the datum's values to what the server operator wants.
-/datum/controller/subsystem/job/proc/load_jobs_from_config()
-	var/toml_file = "[global.config.directory]/jobconfig.toml"
-
-	if(!legacy_mode) // this flag is set during the setup of SSconfig, and all warnings were handled there.
-		var/job_config = rustg_read_toml_file(toml_file)
-
-		for(var/datum/job/occupation as anything in joinable_occupations)
-			var/job_title = occupation.title
-			var/job_key = occupation.config_tag
-			if(!job_config[job_key]) // Job isn't listed, skip it.
-				message_admins(span_notice("[job_title] (with config key [job_key]) is missing from jobconfig.toml! Using codebase defaults.")) // List both job_title and job_key in case they de-sync over time.
-				continue
-
-			// If the value is commented out, we assume that the server operate did not want to override the codebase default values, so we skip it.
-			var/default_positions = job_config[job_key][TOTAL_POSITIONS]
-			var/starting_positions = job_config[job_key][SPAWN_POSITIONS]
-			var/playtime_requirements = job_config[job_key][PLAYTIME_REQUIREMENTS]
-			var/required_account_age = job_config[job_key][REQUIRED_ACCOUNT_AGE]
-			var/required_character_age = job_config[job_key][REQUIRED_CHARACTER_AGE]
-
-			if(default_positions || default_positions == 0) // We need to account for jobs that were intentionally turned off via config too.
-				occupation.total_positions = default_positions
-			if(starting_positions || starting_positions == 0)
```

Comment:
these were replaced with `isnum()` let me know if that's a fuckup, but it seems right to me

Reply:
it doesn't technically cover as much, but I hope to god none was putting strings in here

Reply:
this comment was before i switched it so that validation of values were done on the datum just in case someone wanted to make string-type config (which was the whole reason why I migrated it from the txt/regex format back in September 2022)